src/styles/card.scss

```diff
@@ -0,0 +1,56 @@
+@import 'variables';
+
+.card {
+  width: $card-width;
+  padding: 20px;
+  border: 1px solid #ddd;
+  border-radius: 8px;
+  font-family: $font-family;
```

Comment:
The variable `$font-family` is used here but not defined in this file. Ensure it is defined in the imported 'variables' file or within this file.